### PR TITLE
Add rhel8 content

### LINKED ISSUE #3613: RHEL8 product tree
Need to create RHEL8 product tree to begin building RHEL8 content.

This has been talked about a few times already -- so not new. Just documenting the open action here.

### PR DESCRIPTION
**Pull request by redhatrises**
- Fixes #3613

### CODE REVIEW

**Comment by jan-cerny** [comment]
Hi @redhatrises 

I have noticed that some templates don't have "Red Hat Enterprise Linux 8", eg. shared/templates/template_ANACONDA_mount, shared/templates/template_BASH_audit_rules_login_events etc. Could you take a look at it?

You add more profiles than are shipped now in RHEL 8 public beta, they look like they're copied from RHEL7. Have you tested if the rules in these additional profiles are applicable on RHEL8?

I also have found using `profile_stats.py` that OSPP profile in this PR has only 52% bash coverage, but in OSPP profile shipped in RHEL8 public beta it's 86%. Maybe some "platform" tag missing?

File: rhel8/profiles/C2S.profile [hunk 1]
```diff
@@ -0,0 +1,245 @@
+documentation_complete: true
+
+title: 'C2S for Red Hat Enterprise Linux 7'
```
Comment (shawndwells) [inline rhel8/profiles/C2S.profile]: C2S reflects the baseline for CIA's Commercial Cloud Services (C2S). There isn't a C2S baseline yet and likely won't be for some time. For now, there shouldn't be a RHEL8 C2S profile.

**Comment by redhatrises** [comment]
> On 11/21/18 5:05 AM, Jan Černý wrote: Hi @redhatrises <https://github.com/redhatrises> I have noticed that some templates don't have "Red Hat Enterprise Linux 8", eg. shared/templates/template_ANACONDA_mount, shared/templates/template_BASH_audit_rules_login_events etc. Could you take a look at it?
> @redhatrises would this be a quick fix?

@shawndwells should be fixed already.

**Comment by shawndwells** [comment]
On 11/21/18 1:30 PM, Gabe Alford wrote:
>
>     On 11/21/18 5:05 AM, Jan Černý wrote: Hi @redhatrises
>     <https://github.com/redhatrises> https://github.com/redhatrises I
>     have noticed that some templates don't have "Red Hat Enterprise
>     Linux 8", eg. shared/templates/template_ANACONDA_mount,
>     shared/templates/template_BASH_audit_rules_login_events etc. Could
>     you take a look at it?
>     @redhatrises <https://github.com/redhatrises> would this be a
>     quick fix?
>
> @shawndwells <https://github.com/shawndwells> should be fixed already.
>

yeah, looks like our emails crossed.

here's to hoping the build still works ;)

File: rhel8/CMakeLists.txt [hunk 2]
```diff
@@ -9,7 +9,6 @@ set(DISA_SRG_TYPE "os")
 ssg_build_product(${PRODUCT})
 
 ssg_build_html_table_by_ref(${PRODUCT} "nist")
-ssg_build_html_table_by_ref(${PRODUCT} "cis")
```
Comment (shawndwells) [inline rhel8/CMakeLists.txt]: Can the CIS table still be built, but not have the C2S profile?

e.g. many rules in other profiles (CUI, STIG) have CIS mappings.

Comment (redhatrises) [reply rhel8/CMakeLists.txt]: @shawndwells yes. fixed.

**Comment by shawndwells** [comment]
Going to merge this.

@jan-cerny is 100% correct there are some rough edges (e.g. templates, remediation coverage). If we hold this PR because of those issues then the entirety of that work falls on @redhatrises and we can't collaborate iteratively. And the resulting PR would be to huge to review.

**Pull request merged**